troubadix_config.toml

```
@@ -0,0 +1,55 @@
+# Use Taplo / VS Code Even Better TOML for formatting
+
+title = "Troubadix Plugin Config File"
+
+[check_http_links_in_tags]
+description = "Strings that should be ignored because they contain a valid URL type in a tag"
```

```suggestion
[check_http_links_in_tags]
```
Those two aren't used, right?

---

are not used in the code, functionally just comments. 
The purpose of the description fields was to make the otherwise not very understandable list of random values make more sense. Basically, give a description of what this particular plugin configuration wants and in what format. This way, if the file is only in the VT repo, developers can make adjustments to it without having to look in the plugin's source code.
This can also be done as comments rather than actual toml values. I just feel like the way comments are grayed out makes them less good as nice readable documentation.

---

I'd be in favor of using comments instead, because if a value isn't actually used anywhere, we shouldn't declare it in the first place and misuse them as comments 🤔

---

done in de8d0b7e8b07473600cdfdf1ef6ad8cdf102b88e